admin_guide/managing_networking.adoc

```diff
@@ -417,50 +418,75 @@ $ oc create -f <replication_controller>.json
 oc describe rc <replication_controller>
 ----
 
-[[multicast]]
+[[admin-guide-networking-multicast]]
 == Enabling Multicast
 
-Normally, multicast traffic between pods is disabled in {product-title}, but you
-can enable it on a per-project basis by setting an annotation on the project's
-corresponding NetNamespace object:
-
+[IMPORTANT]
+====
+Enabling multicast networking is a Technology Preview only. Technology
+Preview features are not supported with Red Hat production service level
+agreements (SLAs), might not be functionally complete, and Red Hat does not
+recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, 
+see https://access.redhat.com/support/offerings/techpreview/. 
 ====
+
+Multicast traffic between {product-title} pods is disabled by default. Enable it
+on a per-project basis by setting an annotation on the project's corresponding
+NetNamespace object:
+
 ----
-# oc annotate netnamespace <namespace_name> \
+# oc annotate netnamespace <namespace> \
     netnamespace.network.openshift.io/multicast-enabled=true
 ----
-====
-
-This is a Tech Preview feature and the details may change in the future.
 
-You can later disable multicast again by removing the annotation:
+Disable multicast by removing the annotation:
 
-====
 ----
-# oc annotate netnamespace <namespace_name> \
+# oc annotate netnamespace <namespace> \
     netnamespace.network.openshift.io/multicast-enabled-
 ----
-====
 
-If you have <<joining-project-networks,joined networks together>> with
-`oadm pod-network` you will need to enable multicast in each of the projects'
+If you have
+xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined
+networks together], you will need to enable multicast in each of the projects'
 NetNamespaces in order for it to take effect in any of the projects. You cannot
-enable multicast in the "default" project or in any other project that has been
-<<making-project-networks-global,made global>>, nor can you send multicast
-packets into or out of global projects.
+enable multicast in the `default` project, or in any other project that has been
+xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made
+global], nor can you send multicast packets into or out of global projects.
```

Comment:
Oops, this behavior actually got changed after I originally wrote the docs. Should be something like:

    If you have
    xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined
    networks together], you will need to enable multicast in each of the projects' 
    NetNamespaces in order for it to take effect in any of the projects. Likewise, if
    you want to enable multicast in the `default` project, you must also enable it in
    all other projects that have been
    xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made
    global]. Note that global projects are not "global" with respect to multicast;
    they can communicate via multicast only with other global projects, not with all
    projects in the cluster as with unicast.